NDB.Covid19/NDB.Covid19/Enums/ReportInfectedType.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NDB.Covid19.Enums
+{
+    public enum ReportInfectedType
```

Comment:
Isn't  this enum already defined? Check out `NDB.Covid19.ProtoModels.TemporaryExposureKey.Types`